tensorrt_llm/rlhf_utils.py

```diff
@@ -0,0 +1,66 @@
+from typing import List
```

Comment:
A more generic filename may be better since these functionalities are not strictly only useful for RLHF workflows.